examples/feature_selection/plot_feature_selection.py

```diff
@@ -89,17 +101,35 @@
 svm_weights_selected = np.abs(clf_selected[-1].coef_).sum(axis=0)
 svm_weights_selected /= svm_weights_selected.sum()
 
+# %%
+
+# plot
```

Comment:
I would remove the comments because the labels are indicating the same information

Reply:
Done